<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDialog</class>
 <widget class="QDialog" name="ColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bgBT">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Structure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="structureBT">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="bgASB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="objASB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="applyBT">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="presWT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Selection</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Color</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>R</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Slice</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addPresBT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Pres</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
